src/base/LruMap.h

```diff
@@ -53,9 +66,17 @@ template <class EntryValue, size_t EntryCost = sizeof(EntryValue)> class LruMap
     /// The free space of the map
     size_t freeMem() const { return (memLimit() > size() ? memLimit() - size() : 0);}
     /// The current size of the map
-    size_t size() const {return (entries_ * EntryCost);}
+    size_t size() const {return memUsed_;}
```

Comment:
I'm not going to change the size() name in this PR because it would require altering a lot of code unrelated to the changes required by Bearer tokens. Please feel free to submit a separate PR for that rename.
Renaming EntryCost now though since it is new.